src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/PullRequestPostAnalysisTask.java

```diff
@@ -125,7 +125,7 @@ public void finished(PostProjectAnalysisTask.ProjectAnalysis projectAnalysis) {
                                     postAnalysisIssueVisitor, qualityGate,
                                     new AnalysisDetails.MeasuresHolder(metricRepository, measureRepository,
                                                                        treeRootHolder), analysis,
-                                    projectAnalysis.getProject(), server.getPublicRootUrl());
+                                    projectAnalysis.getProject(), configuration, server.getPublicRootUrl());
```

Comment:
This class is getting too big judging by the number of constructor parameters. There probably needs to be some refactoring of this class, but I'm happy to pick this up at a later point if needed.